crates/webidl/src/lib.rs

```diff
@@ -70,6 +72,45 @@ pub fn compile(webidl_source: &str) -> Result<String> {
     Ok(compile_ast(&ast))
 }
 
+/// ChromeOnly is for things that are only exposed to privileged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &Vec<Box<ExtendedAttribute>>) -> bool {
+    ext_attrs.iter().map(|external_attribute| {
+        let mut ret = false;
+        match external_attribute {
+            box ExtendedAttribute::ArgumentList(al) => {
+                if al.name == "ChromeOnly" {
+                    ret = true;
+                    println!("ArgumentList");
+                }
```

Comment:
You can replace this whole `if` block with `al.name == "ChromeOnly"` (no semicolon) and similarly for the later branches. Doing so will cause the `match` block to evaluate to a `bool`.